etl/steps/data/garden/igh/2024-07-05/better_data_homelessness.meta.yml

```diff
@@ -0,0 +1,310 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    display: &common_display
+      tolerance: 5
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Homelessness
+        - Poverty
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  better_data_homelessness:
+    variables:
+      people_experiencing_homelessness:
+        title: Number of people experiencing homelessness
+        unit: people
+        short_unit: ""
+        description_short: Due to different definitions and data collection methods, the estimated number of people experiencing homelessness is not directly comparable across countries.
+        presentation:
+          title_public: Number of people experiencing homelessness
+        display:
+          name: Number of people experiencing homelessness
+          numDecimalPlaces: 0
+          <<: *common_display
+
+      people_homeless_per_10k:
+        title: People experiencing homelessness per 10,000 people
+        unit: people
```

Comment:
Nit pick but perhaps the unit here should acknowledge that it's a rate? Although maybe hard to do without having a really long unit...

Reply:
I will change it for `people in homelessness per 100,000 population`. Thanks!